src/SeleniumLibrary/__init__.py

```diff
@@ -70,6 +71,24 @@ class SeleniumLibrary(DynamicCore):
     - `Shortcuts`
     - `Keywords`
 
+    = Webdrivers =
+
+    In order for SeleniumLibrary to succesfully interact with browser,
+    a webdriver module needs to be present in the host that is executing the
```

Comment:
Instead of word `module` would a better word be `binary`? Example: `a webdriver binary needs to...` Or `executable` perhaps?

Reply:
i used the word "module" as it was used in introductory pagagraph: 

```SeleniumLibrary uses the Selenium WebDriver modules internally to control a web browser.``` :D